source/jormungandr/jormungandr/street_network/utils.py

```diff
@@ -47,6 +47,10 @@ def crowfly_distance_between(start_coord, end_coord):
     return EARTH_RADIUS_IN_METERS * 2.0 * math.asin(math.sqrt(lat_h + tmp * lon_h))
 
 
+def get_manhattan_duration(distance, speed):
```

Comment:
There are other get_manhattan_duration in jormungandr. Isn't it better to add one in /jormungandr/utils.py and delete other functions and their uses ?

Reply:
It is a good idea.
There is 3 pathes of manhattan function:
- jormun/utils
- scenario/helper_classes/fallback_duration
- jormun/street_network/utils

Maybe in an other PR i don't want to change everywhere in this PR